snyk-monitor-cluster-permissions.yaml

```diff
@@ -87,4 +87,7 @@ data:
   namespace: ""
   integrationApi: ""
   clusterName: ""
+  http_proxy: ""
```

Comment:
any changes required for the OS operator?

Reply:
It's based on the Helm chart so it will automatically pick it up and we don't have to do anything else!